gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -90,9 +91,10 @@ export const wrapCommand = (command) => {
         this.inspectProposalState(proposal, threshold, owners)
         return result
       }
-
       const result = await this.wrapAction(this.executeProposal)(proposal, proposalContext)
-      this.inspectProposalState(proposal, threshold, owners)
+      if (Object.keys(result).length === 0) {
+        logger.error('Execution failed')
+      }
```

Comment:
`wrapAction` should throw if there's an error, which is the only case where this happens: `Object.keys(result).length === 0`
Please remove this
```suggestion
```